pkgs/servers/rt/default.nix

```diff
@@ -82,18 +136,27 @@ stdenv.mkDerivation rec {
     "--enable-graphviz"
     "--enable-gd"
     "--enable-gpg"
-    "--with-db-type=SQLite"
+    "--enable-smime"
+    "--with-db-type=Pg"
```

Comment:
Does this mean that SQLite support is dropped?

Reply:
Hm. I'd be surprised if anyone is actually using this from nixpkgs right now. I see you fixed the build in 3e50e26e7d4c269bbbb33ce7f8b298b4b5477bca. It has broken since. Trying to run the build from that commit doesn't work either, because the `RT_SiteConfig.pm` isn't writable.

Reply:
Good points.